lib/ts/test_Ptr.cc

```diff
@@ -95,5 +95,5 @@ REGRESSION_TEST(Ptr_bool)(RegressionTest *t, int /* atype ATS_UNUSED */, int *ps
   Ptr<PtrObject> some = make_ptr(new PtrObject(&alive));
```

Comment:
I would like this moved to a catch based test as part of the clean up

Reply:
Moving the test is independent of whether or not we want to change Ptr conversion to bool.  Let's make up our minds first about this, then I'll move the test in a separate PR.